### PR TITLE
Adds basic test framework

### PR DESCRIPTION
**Pull request by duncangrant**
Opens some ports that seem necessary

### CODE REVIEW

File: cassandra-catalog.bom [hunk 1]
```diff
@@ -63,12 +63,13 @@ brooklyn.catalog:
...
       brooklyn.config:
         cassandra.archive.url: http://archive.apache.org/dist/cassandra/3.5/apache-cassandra-3.5-bin.tar.gz
+        cql.port: 9042
+        gossip.port: 7000
         provisioning.properties:
           minRam: 4g
 
       shell.env:
         C_URL: $brooklyn:config("cassandra.archive.url")
-        C_CLUSTER_NAME: $brooklyn:config("cassandra.cluster.name")
```
Comment (googlielmo) [inline cassandra-catalog.bom:71]: I believe this deletion was done unintentionally, as the cluster name is one of the few config keys that we support.

**Review by googlielmo — CHANGES_REQUESTED** [review]
Please revert cluster name.
Port openings are correct, thank you for including those, they will be needed in some clouds (e.g., AWS)

File: cassandra-catalog.bom (hunk 1 above)
Comment (duncangrant) [reply cassandra-catalog.bom:71]: Oops - I'll fix

**Comment by googlielmo** [comment]
LGTM. Merging.

**Pull request merged**